packages/flutter_tools/test/general.shard/license_collector_test.dart

```diff
@@ -430,7 +430,7 @@ void main() {
     expect(licenseResult.dependencies, isEmpty);
     expect(licenseResult.errorMessages.single,
       'package foo specified an additional license at /foo.txt, but this file could not be read:'
```

Comment:
I fixed memory file system to throw A FileSystemError instead of a FormatException to match the behavior of the real file system.

Reply:
Reference: https://github.com/google/file.dart/pull/181